docs/calendar-draft.md

```diff
@@ -32,38 +32,38 @@ class MyCalendar {
 	///////////////////
 
 	/** Returns the projection of self in the ISO calendar */
-	[Temporal.Calendar.toISO](
+	toISO(
 		self: Temporal.Date
 	) : Temporal.Date;
 
 	/** Returns the projection of isoDate in the custom calendar */
-	[Temporal.Calendar.fromISO](
+	fromISO(
 		isoDate: Temporal.Date
 	) : Temporal.Date;
 
 	/** A string identifier for this calendar */
-	[Temporal.Calendar.id] : string;
+	id] : string;
```

Comment:
```suggestion
	id : string;
```

altho this should ideally be an accessor on the prototype, rather than an own property?